core/eth/types.go

```diff
@@ -290,3 +290,13 @@ func (ary UntrustedBytes) SafeByteSlice(start int, end int) ([]byte, error) {
 	}
 	return ary[start:end], nil
 }
+
+// ChainlinkEthLogFromGethLog returns a copy of l as an eth.Log. (They have
+// identical fields, but the field tags differ, and the types differ slightly.)
+//
+// A better place for this might be utils, but that creates an import cycle
+func ChainlinkEthLogFromGethLog(l types.Log) Log {
```

Comment:
Pretty sure I already wrote something identical to this already... try `toCLEthLog`

Reply:
I moved this into `cltest.go`. That triggered an import cycle in `vrf`, so I moved all the vrf test code to `vrf_test`, and there are a few dozen minor  changes resulting from that.